.github/workflows/android-maven-release.yml

```diff
@@ -25,10 +25,37 @@ on:
         required: true
         type: string
 
-      constants-paths:
-        description: 'Comma separated string of relative paths to constants files. Example: "code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/OptimizeConstants.java,code/optimize/src/androidTest/java/com/adobe/marketing/mobile/optimize/OptimizeTestConstants.java"'
+      dependencies:
+        description: |
+          A comma-separated list of dependencies with their versions to verify. 
+          Each dependency can optionally specify the files where it applies using the `@` symbol.
+          
+          **Syntax:** `<name> <version>[@file1[,file2,...]]`
+
+          - If the `@` syntax is used, the dependency will only apply to the specified files.
+          - If the `@` symbol is omitted, the dependency applies to all relevant files.
+
+          **Example:**
+          `"AEPCore 3.1.1, AEPServices 8.9.10@AEPCore.podspec, Edge 3.2.1@Package.swift"`
```

Comment:
Can you update this to Android specific examples?

Reply:
Can we remove this from the release file for now? It adds unnecessary clutter to the release workflow.